nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/TestDistributionObject.java

```diff
@@ -65,12 +75,56 @@ public void setup() {
 
     @Test
     public void testDistribution() throws IOException {
+        final String PRODUCT_NAME = "cool product name";
+        final String PRODUCT_VERSION = "cool product version";
+        final String DISTRIBUTION_NAME = "cool distribution name";
+        final String DISTRIBUTION_VERSION = "cool distribution version";
+        final String DISTRIBUTION_SERVER = "cool distribution server";
+        final String DISTRIBUTION_DATE = "cool distribution date";
+
+        Map<String, String> swap = new HashMap<>();
+        swap.put(Environment.PRODUCT_NAME, PRODUCT_NAME);
+        swap.put(Environment.PRODUCT_VERSION, PRODUCT_VERSION);
+        swap.put(Environment.DISTRIBUTION_NAME, DISTRIBUTION_NAME);
+        swap.put(Environment.DISTRIBUTION_VERSION, DISTRIBUTION_VERSION);
+        swap.put(Environment.DISTRIBUTION_SERVER, DISTRIBUTION_SERVER);
+        swap.put(Environment.DISTRIBUTION_DATE, DISTRIBUTION_DATE);
+        swapProperties(swap);
```

Comment:
You could do much simpler here, what we usually do is use the `Framework` properties directly:
```
Map<String, String> properties = new HashMap<>();
properties.put(Environment.PRODUCT_NAME, PRODUCT_NAME);
properties.put(Environment.PRODUCT_VERSION, PRODUCT_VERSION);
...
Framework.getProperties().putAll(properties);
try {
    ...
} finally {
    properties.keySet().stream().forEach(key -> Framework.getProperties().remove(key));
}
```
Note that these properties are `null` to start with so there is no risk of losing one (otherwise you could always fetch them and put/remove only if null).

Reply:
Done.
About JsonAssert, I can't use contains or childrenContains because it expects to have an exact match.
I only used it to assert the node is an array, then kept with my String.contains